samples/demo-authorizationserver/src/main/java/sample/config/AuthorizationServerConfig.java

```diff
@@ -121,9 +121,7 @@ public SecurityFilterChain authorizationServerSecurityFilterChain(
 					new LoginUrlAuthenticationEntryPoint("/login"),
 					new MediaTypeRequestMatcher(MediaType.TEXT_HTML)
 				)
-			)
-			.oauth2ResourceServer(oauth2ResourceServer ->
```

Comment:
This should not be removed as `oauth2ResourceServer()` should be configured with the `authorizationServerSecurityFilterChain()` NOT the `defaultSecurityFilterChain()`.